src/frontend/next/src/components/ScrollAction/ScrollAction.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC, ReactChildren, MouseEvent } from 'react';
+
+type ScrollActionProps = {
+  children: ReactChildren;
+};
+
+const ScrollAction: FC<ScrollActionProps> = ({ children }) => {
```

Comment:
Do we not need to type `{ children } : ScrollActionProps` here?

Reply:
The line `FC<ScrollActionProps>` ensures that any props passed into this component must be of type "ScrollActionProps". So it's already taken cared of here

Reply:
I thought `const ScrollAction: FC<ScrollActionProps>` was seting the return type to `ScrollActionProps`.

Reply:
TS is so confused.

Reply:
@PedroFonsecaDEV This part means ScrollAction is a functional component and it takes props in the form of ScrollActionProps

Reply:
@humphd : We don't need to put ScrollActionProps behind the props because it's already taken care of inside FC